src/shortcuts.js

```diff
@@ -365,7 +365,7 @@ export const globalShortcuts = perma(() => [
         }, RENDER_DELAY)
       }
       else {
-        e.allowDefault()
+        e.preventDefault()
```

Comment:
This is actually an extended event object that does `e.preventDefault` by default and exposes `e.allowDefault` instead. This behavior is probably confusing and should be reverted to the conventional understanding, but for now this means you should leave this code as-is!

Reply:
My bad, will fix this now